db/migrate/20211216110209_add_allow_messages_after_resolved_to_inbox.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllowMessagesAfterResolvedToInbox < ActiveRecord::Migration[6.1]
+  def change
+    add_column :inboxes, :allow_messages_after_resolved, :boolean, default: true
```

Comment:
The current behavior is that if we enable CSAT surveys then it would disable newer messages after resolution. Should we update the script to maintain the same behavior? If CSAT is enabled, then set the value as false, otherwise true. The default can still be true. 

Thoughts @muhsin-k @sojan-official

Reply:
makes sense to me. let's update the migration to toggle value as false for inboxes with CSAT enabled.